lib/rules/no-undef-properties.js

```diff
@@ -111,6 +111,11 @@ module.exports = {
     ).map(toRegExp)
     const propertyReferenceExtractor = definePropertyReferenceExtractor(context)
     const programNode = context.getSourceCode().ast
+    /**
+     * Property names identified as defined via a Vuex or Pinia helpers
+     * @type {string[]}
+     */
+    const propertiesDefinedByVuexOrPiniaHelpers = []
```

Comment:
This could be a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), which would make lookup more performant.

Also please rename to `propertiesDefinedByStoreHelpers` to make it a bit more concise.